src/pages/membership/components/approveYouthProfileForm/ApproveYouthProfileForm.tsx

```diff
@@ -208,13 +207,13 @@ function ApproveYouthProfileForm(props: Props) {
                 labelText={
                   <span className={styles.listLabel}>
                     {t('approval.approveTermsText_1')}{' '}
-                    <Link
-                      to="/terms-of-service"
+                    <a
+                      href={t('registry.descriptionLink')}
                       target="_blank"
                       rel="noopener noreferrer"
                     >
                       {t('approval.approveTermsText_link')}
-                    </Link>
+                    </a>
                     {t('approval.approveTermsText_2')}
                   </span>
```

Comment:
These translation keys still point to the old terms text. I think you could replace these with `Trans` component + `registration.approveTerms`  key.

Reply:
Good point. Hmn, I'll check with whether we can get he new translations into the Google sheet in the same format as the other link.